fe/fe-common/src/main/java/org/apache/doris/common/io/DiskUtils.java

```diff
@@ -46,52 +44,83 @@ public static Df df(String dir) {
             return df;
         }
 
+
         Process process;
         try {
-            process = Runtime.getRuntime().exec("df " + dir);
+            process = Runtime.getRuntime().exec("df -k " + dir);
```

Comment:
What about using `df -h`?
```suggestion
            process = Runtime.getRuntime().exec("df -h " + dir);
```

Reply:
When displayed in the sql result, this will be automatically converted to display in G units. Preserving the accuracy may be used for other purposes later. We can convert it into the format we like at will.

Reply:
Ok